src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -637,6 +637,10 @@ int AbortEnergyPlus(EnergyPlusData &state)
         state.files.flushAll();
     }
 
+    // The audit file seems to be held open in some cases, make sure it is closed before leaving.
+    // EnergyPlus can close through two paths: EndEnergyPlus and AbortEnergyPlus, so do the same thing there.
+    state.files.audit.close();
```

Comment:
This works well, but I do think we need to reconsider our handling of closing files.